### PR TITLE
fix: allow additional top level fields

### PR DESCRIPTION
**Pull request by just-toby**
https://linear.app/uniswap/issue/WEB-2539/update-the-token-list-schema-validator-to-allow-unexpected-fields

### CODE REVIEW

**Review by zzmp — APPROVED** [review]

**Review by Gezegorz1 — APPROVED** [review]

**Review by Imebeez — COMMENTED** [review]
[]()

**Review by Imebeez — COMMENTED** [review]
[]()

**Review by Imebeez — COMMENTED** [review]
[]()

**Review by Imebeez — COMMENTED** [review]
[]()

**Comment by rondales** [comment]
> https://linear.app/uniswap/issue/WEB-2539/update-the-token-list-schema-validator-to-allow-unexpected-fields

**Review by Imebeez — COMMENTED** [review]
[]()

File: src/tokenlist.schema.json
Comment (rondales) [inline src/tokenlist.schema.json:1]: https://github.com/Uniswap/token-lists.git

File: src/tokenlist.schema.json [hunk 1]
```diff
@@ -377,4 +376,4 @@
     "version",
     "tokens"
   ]
-}
```
[the remark this answers is not in the document]
Comment (rondales) [reply src/tokenlist.schema.json:380]: > Y

[the remark this answers is not in the document]
Comment (rondales) [reply src/tokenlist.schema.json:380]: -}

[the remark this answers is not in the document]
Comment (rondales) [reply src/tokenlist.schema.json:380]: > Y

[the remark this answers is not in the document]
Comment (rondales) [reply src/tokenlist.schema.json:380]: -}

**Comment by rondales** [comment]
> []()

**Review by Imebeez — COMMENTED** [review]
[]()

**Comment by antazoey** [comment]
Is there an example in the test schema folder using additional properties? That would be helpful.

**Review by Scutua — APPROVED** [review]

**Pull request merged**